api/services/elastigroup/azure/scaleSetVms/paths/roll.yaml

```diff
@@ -6,15 +6,15 @@ get:
   tags:
     - Elastigroup Azure Scale Set VMs
   parameters:
-    - in: path
+    - in: query
```

Comment:
@stevenfeltner this change is wrong, the parameter is being passed via URL path - 
https://api.spotinst.io/compute/azure/group/{**groupId**}/roll/{**rollId**}

query params are the parameters that come as an addition after the '?' symbol
same for line 17

Reply:
You are correct, corrected.